jsonschema-validation.xml

```diff
@@ -213,6 +213,22 @@
                 <eref target="https://json-schema.org/draft/2020-12/meta/validation"/>.
             </t>
 
+            <section title="Keyword Independence">
+                <t>
+                    Schema keywords typically operate independently, without
+                    affecting each other's outcomes.
+                </t>
+                <t>
+                    For schema author convenience, there are some exceptions among the
+                    keywords in this vocabulary:
+                    <list>
+                        <t>
+                            "requiredAllExcept", whose behavior is defined in terms of "properties"
```

Comment:
If we're going to have a list, we should probably add `minContains` and `contains`, `additionalProperties`, and any others that have dependencies.

Reply:
Currently there is "Keyword Independence" section for each vocabulary that includes keyword independence exceptions. The validation vocabulary didn't have one yet, so I added it. All the other keyword exceptions are mentioned in their respective vocabularies.

Reply:
How does this fit in, then? Are there no other dependent keywords in this vocabulary?

Reply:
`minContains` and `maxContains` are in this vocabulary, so they ought to be listed here.

Reply:
`minContains` and `maxContains` are now included as well.